tools-and-tests/k6/test/bn-query-validation.js

```diff
@@ -1,63 +1,68 @@
 import { Client, StatusOK } from 'k6/net/grpc';
 import { check, sleep } from 'k6';
 import { SharedArray } from 'k6/data';
+import {GetBlockRequest, ServerStatusRequest} from "../lib/grpc.js";
```

Comment:
Q: why the rename on folder.
The gaol was to maintain a directory structure for each types of test. See https://github.com/hiero-ledger/hiero-block-node/tree/main/tools-and-tests/k6#test-types

Smoke tests are intentionally for simple sanity runs